test/factories/teaching_period_factory.rb

```diff
@@ -0,0 +1,9 @@
+FactoryGirl.define do
+  factory :teaching_period do
+    sequence(:period)        { |n| "T#{n}" }
+    start_date               { Faker::Date.between(2.days.ago, Date.today) }
```

Comment:
Change this to use a sequence adding  `n * 15.weeks` to the current date

Reply:
Hi Andrew, I was thinking that the following for start_date would work
`sequence(:start_date)    { |n| Datetime.now + n*15.weeks}`
but it didnt. It throws an error of 'uninitialized constant Datetime'. I also require 'date' package in the factory file but seems not help. Can you suggest any other way?

Reply:
Try `DateTime` not `Datetime`

Reply:
Actually I found out I can use `Time.zone.now` and it's all good 💃 .